scripts/ProcessTemplates.ps1

```diff
@@ -0,0 +1,143 @@
+$mainPath = Split-Path (split-path -parent $MyInvocation.MyCommand.Path) -Parent

```

Comment:
can you add some comments here about what this script does?

it should probably have input checks/etc that echo usage info and stuff if there's no args or whatever (although it looks like it requires no args and just uses the cwd as the arg?